jetty/release_notes.md

```diff
@@ -0,0 +1,116 @@
+{#release-notes}
+
+# Release Notes
+
+## Zenoh in gz-transport
+
+Gazebo now supports Zenoh as an alternative transport implementation, offering
+improved discovery, interoperability, and performance. To enable Zenoh, set the
+environment variable `export GZ_TRANSPORT_IMPLEMENTATION=zenoh`. This allows
+Gazebo to leverage Zenoh's features and potentially integrate more seamlessly
+with ROS 2 and other systems utilizing Zenoh.
+
+See the [tracking issue](https://github.com/gazebosim/gz-transport/issues/559)
+and the implementation pull request
+<https://github.com/gazebosim/gz-transport/pull/665> for more details.
+
+## Remove major version from package names
+
+We have refactored package naming to remove major version numbers, simplifying
+dependency management and reducing maintenance overhead. This change impacts how
+Gazebo packages are identified in CMake builds, package installations, and
+import statements.
+
+See the
+[tracking issue](https://github.com/gazebo-tooling/release-tools/issues/1244)
+for more details.
+
+## Migrate from Qt5 to Qt6
+
+With Qt5 reaching
+[end of life](https://www.qt.io/blog/extended-security-maintenance-for-qt-5.15-begins-may-2025),
+Gazebo has been migrated to Qt6. While there are no major changes to end-users
+of the Gazebo GUI, `gz-gui` plugin authors will need to make changes to ensure
+compatibility with Qt6. Make sure to check out the
+[migration guide](https://gazebosim.org/api/gui/10/migration_qt6.html).
+
+See the [tracking issue](https://github.com/gazebosim/gz-gui/issues/586) and the
+implementation pull requests <https://github.com/gazebosim/gz-gui/pull/666> and
+<https://github.com/gazebosim/gz-sim/pull/2832> for more details
+
+## Implement ROS standard simulation interfaces
+
+Gazebo now supports the standard ROS 2
+[simulation interfaces](https://github.com/ros-simulation/simulation_interfaces),
+enabling seamless integration with ROS 2 ecosystems and providing access to
+features for controlling and querying the simulation state. Documentation has
+also been added to guide users on utilizing these new interfaces.
+
+See the [tracking issue](https://github.com/gazebosim/ros_gz/issues/732) and the
+implementation pull requests <https://github.com/gazebosim/ros_gz/pull/790> and
+<https://github.com/gazebosim/docs/pull/601> for more details
+
+## Resolve auto inertia based on input mass
+
+Automatically computed inertial can now use the `mass` specified in the SDF file
```

Comment:
```suggestion
Inertia parameters can now be automatically set based on an object's mass specified in an SDF file. This  
eliminates the need to manually compute and set the density of collision objects. When both mass
```
This is a little awkward and uclear. Perhaps this will help.